### PR TITLE
Edit WiFi Manager build configuration

### PR DESCRIPTION
**Pull request by dr-venkman**
Edit the Kconfig for wifi manager, and carry out the following changes:
1. create choice config for supplicant library, and move the WPA_SUPPLICANT config creation to wifi_manager
2. create choice config for wifi driver, and move the SCSC_WLAN config creation to wifi_manager.

Additionally, document these changes to the docs file for WiFi.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: framework/src/wifi_manager/Kconfig [hunk 1]
```diff
@@ -7,11 +7,34 @@ config WIFI_MANAGER
...
-	select SCSC_WLAN
     ---help---
         Easy APIs for applications to use and control Wi-Fi features
 
+choice
+	prompt "WiFi library"
+	default WPA_SUPPLICANT
+	depends on WIFI_MANAGER
+
+config WPA_SUPPLICANT
```
Comment (sunghan-chang) [inline framework/src/wifi_manager/Kconfig]: WPA_SUPPLICANT and SCSC_WLAN should be each folder, not here.

How about below?
```
config ARCH_HAVE_WIFI_DRIVER
	bool

config ARCH_HAVE_WPA_SUPPLICANT
	bool

config WIFI_MANAGER
	bool "Enable Wi-Fi Manager"
	default n
	select NETUTILS_WIFI
	select WPA_SUPPLICANT		if !ARCH_HAVE_WPA_SUPPLICANT
	select SCSC_WLAN		if !ARCH_HAVE_WIFI_DRIVER
	select WIFI_3RD_PARTY_FIRMWARE	if ARCH_HAVE_WPA_SUPPLICANT || ARCH_HAVE_WIFI_DRIVER
	---help---
		 Easy APIs for applications to use and control Wi-Fi features

config WIFI_3RD_PARTY_FIRMWARE
	bool

if WIFI_3RD_PARTY_FIRMWARE
# Do it something
endif
```

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by sunghan-chang** [comment]
We need to modify two things because you modify this.
1. Let's set a default vale of wpa_supplicant to y. Because that will be called only when that is needed.
2. SELECT_SCSC_WLAN does not wrap SCSC_WLAN like SELECT_WPA_SUPPLICANT.

**Comment by dr-venkman** [comment]
Will make the change for suggestion 1. Not sure I understand suggestion 2. If you mean to source the Kconfig for wireless driver, then it is currently done already in os/drivers/Kconfig. Does that need to change, if we go by suggestion 2?

**Comment by sunghan-chang** [comment]
Let me explain this face to face.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by sunghan-chang** [comment]
Looks good to me. But you had better ask a merge to Pilseob or Sangwon who is in charge of network (wifi).

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check Result:
lib/libc/math/lib_gamma.c:93: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:93: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
lib/libc/math/lib_gamma.c:94: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:95: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:95: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
lib/libc/math/lib_gamma.c:96: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:96: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:97: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:97: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:98: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:98: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:99: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:99: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:100: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:100: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:101: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:101: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
lib/libc/math/lib_gamma.c:102: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:102: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:103: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:103: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:104: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:104: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:105: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:105: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:106: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:106: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:107: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:107: ERROR: [BRC_M_SMT] that open brace { should be on the previous line
lib/libc/math/lib_gamma.c:108: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:108: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:109: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:109: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:110: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:110: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:111: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:111: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:112: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libc/math/lib_gamma.c:112: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:113: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:114: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libc/math/lib_gamma.c:114: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
lib/libc/wchar/lib_wcslcpy.c:99: ERROR: [BRC_M_SMT] while should follow close brace '}'
lib/libxx/libcxx/include/atomic_support.h:34: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:34: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:35: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:35: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:36: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:36: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:37: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:37: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:38: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:38: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:39: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:39: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:40: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:40: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:41: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:41: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:42: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:42: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:50: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:63: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:64: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:65: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:66: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:67: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:68: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:71: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:71: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:73: ERROR: [SPC_M_OPR] need consistent spacing around '*' (ctx:VxW)
lib/libxx/libcxx/include/atomic_support.h:74: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:74: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:76: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:79: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:79: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:81: ERROR: [SPC_M_OPR] need consistent spacing around '*' (ctx:VxW)
lib/libxx/libcxx/include/atomic_support.h:83: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:86: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:86: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:89: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:89: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:91: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:94: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:94: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:97: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:97: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:99: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:102: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:102: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:105: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:106: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:107: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:109: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:110: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:110: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:116: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:117: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:118: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:119: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:120: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:121: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:124: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:124: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:127: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:127: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:129: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:132: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:132: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:136: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:139: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:139: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:142: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:142: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:144: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:147: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:147: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:150: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:150: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:152: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:155: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:WxV)
lib/libxx/libcxx/include/atomic_support.h:155: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
lib/libxx/libcxx/include/atomic_support.h:158: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:159: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:161: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:162: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:162: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:163: ERROR: [IDT_M_TAB] code indent should use tabs where possible
lib/libxx/libcxx/include/atomic_support.h:163: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:164: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:165: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
lib/libxx/libcxx/include/atomic_support.h:166: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/ctype.h:88: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/ctype.h:89: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/ctype.h:296: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/ctype.h:297: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/ctype.h:298: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/ctype.h:316: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/ctype.h:334: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
os/include/libcxx/__bsd_locale_fallbacks.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
os/include/libcxx/__bsd_locale_fallbacks.h:39: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
os/include/libcxx/__bsd_locale_fallbacks.h:39: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxO)
os/include/libcxx/__bsd_locale_fallbacks.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxO)
os/include/libcxx/__bsd_locale_fallbacks.h:39: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
os/include/libcxx/__bsd_locale_fallbacks.h:40: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
os/include/libcxx/__bsd_locale_fallbacks.h:40: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
os/include/libcxx/__bsd_locale_fallbacks.h:40: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
os/include/libcxx/__bsd_locale_fallbacks.h:40: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
os/include/libcxx/__bsd_locale_fallbacks.h:45: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:45: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:45: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:46: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:52: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:52: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:52: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:53: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:59: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:59: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:59: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:60: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:65: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/libcxx/__bsd_locale_fallbacks.h:65: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:67: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:67: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:67: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:68: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:74: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:74: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:74: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:75: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:80: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/libcxx/__bsd_locale_fallbacks.h:80: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:82: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:82: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:82: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:83: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:88: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/libcxx/__bsd_locale_fallbacks.h:88: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:90: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:90: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:90: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:91: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:97: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:97: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:97: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:98: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:104: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:104: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:104: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:105: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:111: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:111: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:111: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:112: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:117: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/libcxx/__bsd_locale_fallbacks.h:117: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:119: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:119: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:119: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:120: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:124: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
os/include/libcxx/__bsd_locale_fallbacks.h:125: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:126: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:127: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:127: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:127: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:128: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:129: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:130: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:134: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
os/include/libcxx/__bsd_locale_fallbacks.h:135: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:136: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:137: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:137: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:137: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:138: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:139: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:140: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:144: ERROR: [BRC_M_FTN] open brace '{' following function declarations go on the next line
os/include/libcxx/__bsd_locale_fallbacks.h:145: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:146: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:147: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:147: ERROR: [SPC_M_SEP] space prohibited after that open parenthesis '('
os/include/libcxx/__bsd_locale_fallbacks.h:147: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/libcxx/__bsd_locale_fallbacks.h:148: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:149: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/libcxx/__bsd_locale_fallbacks.h:150: ERROR: [IDT_M_TAB] please, no spaces at the start of a line

**Comment by sunghan-chang** [comment]
```Merge branch 'master' into edit_wifi_manager_build``` is not allowed. Please remove it.

**Pull request closed without merging**